frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1639,7 +1639,7 @@
             };
             parameters.callback = {
                 onSuccess: function() {
-                    $rootScope.notify("success", "Evaluation worker resources scaled successfully");
+                    $rootScope.notify("success", "Evaluation worker resources scaled successfully. Please wait until the old worker is stopped and the new worker starts before making further changes.");
```

Comment:
Change message to `Evaluation worker resources scaled successfully.`

Reply:
Done